lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -510,6 +510,16 @@
       name: customimage001
       resource_group: myResourceGroup
 
+- name: Create a VM with a image id
+  azure_rm_virtualmachine:
+    resource_group: myResourceGroup
+    name: testvm001
+    vm_size: Standard_DS1_v2
+    admin_username: adminUser
+    admin_password: password01
+    image:
+      id: '/subscriptions/<our-subscription-id>/resourceGroups/<our-shared-image-gallery-rsg>/providers/Microsoft.Compute/galleries/<our-gallery-name>/images/rhel-7-cis/versions/1.0.1'
```

Comment:
could you pls rename the variables in the id to like mySubscriptionId,  myResourceGroup,  myGallery etc for code consistency? thanks.

Reply:
<our-subscription-id> -> xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx

to be consistent with other modules